install.sh

```diff
@@ -122,7 +123,14 @@ function extract_install () {
     echo "Extracting ${1} (this may take a while)..."
     tar xpf ../"${2}"
     echo "Installing ${1} (this may take a while)..."
-    tar cpf - ./*/* | (cd /; tar xp --keep-directory-symlink -f -)
+    local TarPaths=()
+    for folder in $crew_folders; do
+      if [ -d ."${CREW_PREFIX}"/"$folder" ]; then
+        TarPaths+=(".${CREW_PREFIX}/${folder}")
```

Comment:
Can we make all folders at first? This will be more simple :)

Reply:
(we can add it to `# prepare directories` part)

Reply:
The point was to only do it to folders which have been chowned, otherwise you might get a permissions error. This protects from that!